server/models/contractor.js

```diff
@@ -0,0 +1,31 @@
+//contractor: skills, jobs completed.  Proposals: contractor
+//name, cost estimate, start date estimate, time frame estimate
+
+const Base = require('./index');
+
+const Contractor = Base.discriminator('Contractor', new mongoose.Schema({
+    skills: {
+        
```

Comment:
Think my comment on skills for Customer applies here too. Think we talk with front end team on how we want to handle input and that will direct us on how we want to handle in the model.